pipeline_steps/common.groovy

```diff
@@ -24,8 +24,13 @@ def install_ansible(){
     # UG-613 change TMPDIR to directory with more space
     export PREV_TMPDIR=\$TMPDIR
     export TMPDIR="/var/lib/jenkins/tmp"
-    # These pip commands cannot be combined into one.
-    pip install --index-url https://pypi.python.org/simple pip==9.0.1 setuptools==33.1.1
+
+    # UG-612 Installing pip and other packages necessary
+    CURL_CMD="curl --silent --show-error --retry 5"
+    OUTPUT_FILE="get-pip.py"
+    \${CURL_CMD} https://bootstrap.pypa.io/get-pip.py > \${OUTPUT_FILE} ||\
+    \${CURL_CMD} https://raw.githubusercontent.com/pypa/get-pip/master/get-pip.py > \${OUTPUT_FILE}
+    python \${OUTPUT_FILE} --isolated pip setuptools wheel
     pip install --isolated -c rpc-gating/constraints.txt six packaging appdirs
```

Comment:
The installation of ``six packaging appdirs`` no longer needs to be seperate. That can be included in the ``requirements.txt`` with the versions we want pinned in ``constraints.txt``